static/src/js/schemas/uiSchemas/collections/temporalInformation.js

```diff
@@ -111,6 +122,36 @@ const temporalInformationUiSchema = {
           }
         ]
       },
+      TemporalResolution: {
```

Comment:
Selecting Contant or Varies Resolution and then filling out Precision of Seconds does not trigger and error asking user to fill in Unit.